config/pmd/ruleset.xml

```diff
@@ -62,7 +67,12 @@
   </rule>
   <rule ref="category/java/design.xml/TooManyMethods">
     <properties>
-      <property name="maxmethods" value="20" />
+      <property name="maxmethods" value="25" />
```

Comment:
Instead of changing the rules is it possible to refactor our code to fulfil this rule?

Reply:
refactored Aravind